docs/rewards/managing-rewards.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 2
+---
+
+# Managing Rewards 
+
+This guide will go through how rewards are managed on the Archway Network by explaining the important fields and operations needed to track and receive rewards. 
+
+## Contract Metadata 
+Contract Metadata is the metadata that instructs the Archway Network who owns the contract and where rewards should be sent. The metadata has three specific parameters: 
+
+`contract_address` - The address of the deployed contract.
+
+`owner_address` - The address of the contract's owner. This field can be both an account or contract address.
+
+`rewards_address` - The account address that will receive the contract's rewards. This field can be both an account or contract address. Tokens are sent to this address after executing a [withdrawal function](#withdrawing-awards) is executed. If this field is not set, the contract will not receive rewards. 
+
+**All above addresses are bech32-encoded.**
+
+### Setting The Contract Metadata 
+
+The contract metadata must be set up by using the `MsgSetContractMetadata` transaction which a contract admin must sign.
+
+```json  
+message ContractMetadata { 
+  string contract_address =  "archway14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sy85n2u";
+  string owner_address = "archway12reqvcenxgv5s7z96pkytzajtl4lf2epyfman2";
+  string rewards_address = "archway12reqvcenxgv5s7z96pkytzajtl4lf2epyfman2";
+}
+``` 
```

Comment:
This is not a valid JSON

Reply:
Good call, I pulled this from the spec which is using proto3 syntax so that is why I used Go before. Can you send me exactly what this is in JSON?

Reply:
```suggestion
```json  
{ 
  "contract_address": "archway14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sy85n2u",
  "owner_address": "archway12reqvcenxgv5s7z96pkytzajtl4lf2epyfman2",
  "rewards_address": "archway12reqvcenxgv5s7z96pkytzajtl4lf2epyfman2"
}
``` 
```

Reply:
Thanks!